instrumentation/log4j/log4j-2.13.2/library/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+dependencies {
+  compileOnly("io.opentelemetry:opentelemetry-sdk-logs")
+
+  library("org.apache.logging.log4j:log4j-core:2.13.2")
```

Comment:
As nobody in its right mind is going to use this version of log4j, should we just start with `2.16`? :)

Reply:
ya, I think this is the only instrumentation that targets a "patch" version too, will do this in follow-up 👍